daml-script/export/it/src/test/resources/interface/Export.daml

```diff
@@ -24,9 +21,12 @@ lookupParty old parties =
 -- | Allocates fresh parties from the party management service.
 allocateParties : Script Parties
 allocateParties = DA.Traversable.mapA allocateParty (DA.TextMap.fromList
-  [ ("Alice", "Alice")
-  , ("Bob", "Bob")
-  , ("Charlie", "Charlie") ])
+  [ ("Alice",
+    "Alice")
+  , ("Bob",
+    "Bob")
+  , ("Charlie",
+    "Charlie") ])
```

Comment:
Those appear on several line because they are actually longer, they are actually of the from
"Party_xxxxxx:xxxxxx" where the xxx are some random string. We replace them by them by a stable reference before comparing the file.